src/main/java/analysis/GoblintAnalysis.java

```diff
@@ -111,10 +122,23 @@ private void preAnalyse() {
     }
 
 
+    private void abortAnalysis() {
+        Process goblintProcess = goblintServer.getGoblintRunProcess().getProcess();
+        int pid = Math.toIntExact(goblintProcess.pid());
+        UnixProcess unixProcess = new UnixProcess(pid);
+        try {
+            unixProcess.kill(2);
```

Comment:
This magic 2 should be extracted to a constant like `SIGINT`, so it's clear which signal it represents.